test/OrchardCore.Tests/Apis/Context/SiteContext.cs

```diff
@@ -81,6 +81,10 @@ public virtual async Task InitializeAsync()
                 TenantName = tenantName;
             }
 
+            // Search indexes are no longer updated in a deferred task at the end of a shell scope
+            // but in a background job after the http request, so they are not already up to date.
+            await Task.Delay(2_000);
```

Comment:
This can make the test flaky. Instead, explicitly wait for the process to finish, don't really on occasionally correct time.

See how you fixed this under another PR of yours when I pointed it out.

Also in `RecentBlogPostsQueryTests`.